test/tests/ver-1b/comparison_ver-1b.py

```diff
@@ -83,7 +88,7 @@
 D = 1
 analytical_flux = C_o * np.sqrt(D/(np.pi * analytical_time)) * \
                   np.exp( x / (2 * np.sqrt(D * analytical_time)))
-ax.plot(tmap_time,tmap_flux,label=r"TMAP8",c='tab:gray')
+ax.plot(tmap_time[1:],tmap_flux[1:],label=r"TMAP8",c='tab:gray')
```

Comment:
Is the [1:] really needed? Why?

Reply:
In this case, the first data point in tmap_flux is zero and the first one in analytical_flux is infinity, as shown in here. I removed it to avoid the large difference. But I think I can add [1:] for analytical plotting to ensure consistency. 
![ver-1b_comparison_flux](https://github.com/user-attachments/assets/5d826207-81d4-4dc0-b680-c74ee6342355)

Reply:
Ok, I get it, it's good with your correction.